### PR TITLE
add:  selectable and draggable behavior

### PR DESCRIPTION
**Pull request by alsturm**

### CODE REVIEW

**Comment by taulinger** [comment]
@alsturm  Does the property "user-drag"  really exist?

File: src/modules/layerManager/components/LayerManager.js [hunk 1]
```diff
@@ -178,9 +178,9 @@ export class LayerManager extends MvuElement {
 			<style>
 				${css}
 			</style>
-			<div class="layermanager overflow-container">
+			<div class="layermanager overflow-container draggable">
```
Comment (thiloSchlemmer) [inline src/modules/layerManager/components/LayerManager.js:181]: should not be draggable, i guess...
this makes the hole container draggable, which includes the action buttons (['expand all'], ['remove all'])

File: src/modules/layerManager/components/LayerManager.js [hunk 2]
```diff
@@ -178,9 +178,9 @@ export class LayerManager extends MvuElement {
...
 				<div class="title">${translate('layerManager_title')}</div>
-				<ul class="layers">
+				<ul class="layers draggable">
```
Comment (thiloSchlemmer) [inline src/modules/layerManager/components/LayerManager.js:183]: the draggable attribute in line 189 should be sufficient, for a full drag&drop-funcitonality of the layers, or not?

I've tested with Chrome and FF so far

**Review by taulinger — APPROVED** [review]
@alsturm  @thiloSchlemmer: Nicely done!

**Pull request merged**